content/influxdb/v2.0/upgrade/v1-to-v2.md

```diff
@@ -12,33 +12,175 @@ aliases:
+
+`influxd upgrade` migrates existing 1.x users and their permissions.
+However, *administrative users are not migrated*.
+
+With the InfluxDB 1.x CLI, perform a `show users` query.
+Any users labeled "admin" will *not* be migrated.
+
+If using an admin user for vizualition or Chronograf administrative functions, you may want to create a new read only user before upgrading.
+Admin rights are granted to the primary user created in the 2.0 setup process via the `influxd upgrade` command.
+(This provides you with the opportunity to re-assess who should be granted admin level access as a part of your InfluxDB 2.0 setup.)
+
+### Dashboards
+
+You can continue to use your existing dashboard/visualization tools with InfluxDB 2.0 via the [1.x read compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/).
+[DBRP mappings](/influxdb/v2.0/reference/api/influxdb-1x/dbrp/) are created as part of the upgrade process to ensure existing users can execute queries with the appropriate permissions.
+Reconfigure your data sources to point to your new InfluxDB 2.0 OSS instance with the appropriate security credentials.
+
+However, if your dashboard tool is configured using a user with admin permissions,
+you will need to create a new read-only user with the appropriate database permissions *before* upgrading.
+This new username/password combination should be used within the data source configurations to continue to provide read-only access to the underlying data.
+
+Ensure your dashboards are all functioning before upgrading.
+
+### Other data
 
-{{% note %}}
 The 1.x `_internal` database is not migrated with the `influxd upgrade` command.
-{{% /note %}}
 
-1. [Download InfluxDB OSS 2.0](https://portal.influxdata.com/downloads/),
-   unpackage the InfluxDB binaries, and then place them in your `$PATH`.   
+### Secure by default
+
+Running InfluxDB with the setting
+
+```
+auth-enabled = false
+```
+
+is no longer supported in InfluxDB 2.0.
+By default authentication is required.
+After upgrading, authentication errors can occur if the username/password credentials within InfluxDB do not match what is currently being used by the various applications, agents, visualization tools, etc.
+Consider enabling authentication *before* upgrading to ensure that you have the appropriate credentials in place.
+
+### Available operating system, container, and platform support
+
+InfluxDB 2.0 is currently available for MacOS and Linux.
+Docker images, ARM builds, and Windows builds are not currently available.
+These are planned for subsequent releases.
+
+There is no plan for a 32-bit build of InfluxDB 2.0.
+
+## Perform the upgrade
+
+If you've considered the guidance above in [Before you begin](#before-you-begin-important-considerations) and are ready to proceed, follow these steps to upgrade your InfluxDB 1.x to InfluxDB 2.0.
+
+{{% warn %}}
+*After* stopping your InfluxDB 1.x process, make a back-up copy of all 1.x data before upgrading with `influx upgrade`.
```

Comment:
isn't this step 2?  stop and copy the directory containing the current data and wal dirs... you will need to refer to the current influxdb.conf to confirm the location of this files.  Example:

Reply:
Ah yes, could do I that way instead of a warning.